src/components/Avatar/index.js

```diff
@@ -42,6 +42,8 @@ Avatar.propTypes = {
     className: PropTypes.string,
     /** An object with custom style applied to the outer element. */
     style: PropTypes.object,
+    /** A valid color. */
```

Comment:
@TahimiLeonBravo this description will need more explanation here it's purpose. it will be fine if we merge the PR with the current text or please provide desired text for @rafaelmg83 here

Reply:
also @TahimiLeonBravo we should consider name this prop backgroundColor since it could be easily related to CSS color property when it will affect/change the background color of the avatar when initials are being used

Reply:
@rafaelmg83 let's change the name of the prop to `backgrondColor`

Reply:
done!